project.clj

```diff
@@ -28,7 +29,10 @@
   :repl-options {:port 41234}
   :min-lein-version "2.0.0"
   :resource-paths ["config" "resources"]
-  :profiles {:dev {:dependencies [[io.pedestal/pedestal.service-tools "0.5.3"]]
+  :profiles {:dev {:dependencies [[io.pedestal/pedestal.service-tools "0.5.3"]
+                                  [com.stuartsierra/component.repl "0.2.0"]
+                                  [org.clojure/tools.namespace "0.2.11"]
```

Comment:
Use tools.namespace 0.3.0-alpha4, it's been the 'good one' for a few years despite the alpha status

(it's a stupid intricacy, you're not at fault)